README.md

```diff
@@ -1,4 +1,52 @@
-# Naginator Plugin for Jenkins
-This plugin allows you to automatically reschedule a build after a build failure
+# Naginator Plugin
 
-More info is available on the plugin's [Wiki page](https://wiki.jenkins-ci.org/display/JENKINS/Naginator+Plugin)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/naginator)](https://plugins.jenkins.io/naginator)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/naginator-plugin?label=changelog)](https://github.com/jenkinsci/naginator-plugin/blob/master/CHANGELOG.md)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/naginator?color=blue)](https://plugins.jenkins.io/naginator)
+[![GitHub contributors](https://img.shields.io/github/contributors/jenkinsci/naginator-plugin?color=blue)](https://github.com/jenkinsci/naginator-plugin/graphs/contributors)
```

Comment:
Badges work correct in plugin.jenkins.io.
Like this: https://plugins.jenkins.io/workflow-support/